src/java.base/share/classes/sun/security/x509/CertAttrSet.java

```diff
@@ -28,57 +28,17 @@
 import sun.security.util.DerOutputStream;
 
 import java.io.IOException;
-import java.security.cert.CertificateException;
 
 /**
- * This interface defines the methods required of a certificate attribute.
- * Examples of X.509 certificate attributes are Validity, Issuer_Name, and
- * Subject Name. A CertAttrSet may comprise one attribute or many
- * attributes.
- * <p>
- * A CertAttrSet itself can also be comprised of other sub-sets.
- * In the case of X.509 V3 certificates, for example, the "extensions"
- * attribute has subattributes, such as those for KeyUsage and
- * AuthorityKeyIdentifier.
- *
- * @author Amit Kapoor
- * @author Hemma Prafullchandra
- * @see CertificateException
+ * This interface defines a certificate attribute that can be DER-encoded.
  */
-public interface CertAttrSet<T> {
+public interface CertAttrSet {
```

Comment:
Should we rename this to "CertAttr" since I think it now supports a single attribute?